style.css

```diff
@@ -0,0 +1,151 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+body {
+  width: 375px;
+  height: 718px;
+  margin: auto;
+  background-color: #344563;
+  font-family: "Poppins", sans-serif;
+}
+
+.header {
+  width: 375px;
+  height: 48px;
+  top: 44px;
+  position: absolute;
+  background-image: url("Media/header.png.jpg");
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+  top: 10px;
+  left: 24px;
+  font-weight: 700px;
+  font-size: 18px;
+  line-height: 20px;
+  align-items: center;
+}
```

Comment:
- [ ] Kindly adhere to the design as much as possible, as I can see you are not using the correct `font-family`. Please keep in mind that as a developer it is very **important** to have **detail-oriented** mindset because it will help you to make a better *developer* who always takes care of small details to make the design looks *professional* and *consistent*. Please have a look at the below table which will help you to solve issue without any doubt.

<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-04-28 22-32-12](https://user-images.githubusercontent.com/66476329/235441314-364acd8f-55cf-4ab5-a44d-0e468e666935.png)

</td>
<td>

![Screenshot from 2023-05-01 14-55-30](https://user-images.githubusercontent.com/66476329/235441279-6852a144-5321-4a04-8533-75a8d5bab751.png)

</td>
</tr>
</table>